src/workloads/encrypted/medical_workload-load-unencrypted.yml

```diff
@@ -0,0 +1,102 @@
+SchemaVersion: 2018-07-01
+Owner: "@10gen/server-security"
+Description: |
+  Models the Queryable Encryption acceptance criteria workloads
+
+
+Clients:
+  EncryptedPool:
+    QueryOptions:
+      maxPoolSize: 400
+
+
+Actors:
+  - Name: InsertActor
+    Type: CrudActor
+    Threads: 16
+    Database: genny_qebench2
+    ClientName: EncryptedPool
+    Phases:
+    - Repeat: 1
+      Collection: {^Parameter: {Name: "Collection", Default: "medical"}}
+      MetricsName: "createIndex"
+      Operations:
+      - OperationName: createIndex
+        OperationCommand:
+          Keys:
+            states: 1
+          IndexOptions: {}
+    - Repeat: 1
+      Collection: {^Parameter: {Name: "Collection", Default: "medical"}}
+      MetricsName: "createIndex"
+      Operations:
+      - OperationName: createIndex
+        OperationCommand:
+          Keys:
+            diagnosis: 1
+          IndexOptions: {}
+    - Repeat: 1
+      Collection: {^Parameter: {Name: "Collection", Default: "medical"}}
+      MetricsName: "createIndex"
+      Operations:
+      - OperationName: createIndex
+        OperationCommand:
+          Keys:
+            status: 1
+          IndexOptions: {}
+    - Repeat: 1
+      Collection: {^Parameter: {Name: "Collection", Default: "medical"}}
+      MetricsName: "createIndex"
+      Operations:
+      - OperationName: createIndex
+        OperationCommand:
+          Keys:
+            credit_cards: 1
+          IndexOptions: {}
+    - Repeat: 1
+      Collection: {^Parameter: {Name: "Collection", Default: "medical"}}
+      MetricsName: "createIndex"
+      Operations:
+      - OperationName: createIndex
+        OperationCommand:
+          Keys:
+            guid: 1
+          IndexOptions: {}
+    - Repeat: 62500
+      Collection: {^Parameter: {Name: "Collection", Default: "medical"}}
+      MetricsName: "load"
+      Operations:
+      - OperationName: insertOne
+        OperationMetricsName: inserts
+        OperationCommand:
+          Document:
+            LoadConfig:
+              Path: ../contrib/qe_test_gen/maps_medical.yml
+              Key: document_insert_medical
+              Parameters:
+                Database: ignored
+                multiplier: 62500
+  
+  - Name: LoggingActor0
+    Type: LoggingActor
+    Threads: 1
+    Phases:
+      - Phase: 0..5
+        LogEvery: 5 minutes
+        Blocking: None
+
+# TODO PERF-3954: Re-enable this test.
+# AutoRun:
+# - When:
+#     mongodb_setup:
+#       $eq:
+#       - shard-lite-fle
+#     branch_name:
+#       $neq:
+#       - v4.0
+#       - v4.2
+#       - v4.4
+#       - v5.0
+#       - v6.0
+#       - v6.1
+#       - v6.2
```

Comment:
no v6.3?

Reply:
Hah. I'd just copied this from the other tests. OK. I've reworked this to use `$gte: 7.0`. How does that look?

Reply:
lgtm!